internal/controller/users_test.go

```diff
@@ -56,19 +56,26 @@ var defaultApiKeyDtoResponse = communication.ApiKeyDtoResponse{
 	ValidUntil: time.Date(2024, 05, 05, 21, 32, 55, 651387237, time.UTC),
 }
 
-func TestUserEndpoints_GeneratesExpectedRoutes(t *testing.T) {
-	assert := assert.New(t)
-
-	actualRoutes := make(map[string]int)
-	for _, r := range UserEndpoints(&mockUserService{}) {
-		actualRoutes[r.Method()]++
+func Test_UsersEndpoints(t *testing.T) {
+	s := RouteTestSuite{
+		generateRoutes: func() rest.Routes {
+			return UserEndpoints(&mockUserService{})
+		},
+		expectedRoutes: map[string]int{
+			http.MethodPost:   3,
+			http.MethodGet:    2,
+			http.MethodPatch:  1,
+			http.MethodDelete: 2,
+		},
+		expectedPaths: map[string]int{
+			"/":             2,
+			"/:id":          3,
+			"/sessions":     1,
+			"/sessions/:id": 2,
```

Comment:
The paths are now correct: the prefix takes care of the `users` part and here we just add the routes at the root level.